modules/rosa-accessing-your-cluster.adoc

```diff
@@ -121,18 +122,21 @@ Created:     May 27, 2020
 +
 [source,terminal]
 ----
-$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443
+$ oc login --token=z3sgOGVDk0k4vbqo_wFqBQQTnT-nA-nQLb8XEmWnw4X --server=https://api.rh-rosa-test-cluster1.j9n4.s1.devshift.org:6443 #<1>
 ----
+<1> For {hcp-title} cluster, the port number should be `443`.
 +
 .Example output
 [source,terminal]
 ----
-Logged into "https://api.rh-rosa-cluster1.j9n4.s1.devshift.org:6443" as "rh-rosa-test-user" using the token provided.
+Logged into "https://api.rh-rosa-cluster1.j9n4.s1.devshift.org:6443" as "rh-rosa-test-user" using the token provided. #<1>
 
 You have access to 67 projects, the list has been suppressed. You can list all projects with 'oc projects'
 
 Using project "default".
 ----
+<1> For {hcp-title} cluster, the port number should be `443`.
```

Comment:
Should this read as "For {hcp-title} clusters" or "For a {hcp-title-first} cluster"?

Might also consider rephrasing for clarity. Something like: "For {hcp-title} clusters, use port 443."

Reply:
Rephrased as, "For a {hcp-title} cluster, use port `443`."